src/XMLexport.cpp

```diff
@@ -648,6 +720,19 @@ bool XMLexport::writeTrigger( TTrigger * pT )
     writeTextElement( "colorTriggerFgColor", pT->mColorTriggerFgColor.name() );
     writeTextElement( "colorTriggerBgColor", pT->mColorTriggerBgColor.name() );
 
+// TODO: The next bit could be revised for a new - not BACKWARD COMPATIBLE form
```

Comment:
This will have to wait until we advance to Mudlet XML format version `1.001` or greater - whenever we feel we can afford to break previous Mudlet versions but then we will want to (and can) provide a downgrade capability in the same manner as we _have_ coded for in the Map file format now!